f5_openstack_agent/lbaasv2/drivers/bigip/lbaas_builder.py

```diff
@@ -49,36 +51,54 @@ def __init__(self, conf, driver, l2_service=None):
             self.service_adapter
         )
         self.l7service = l7policy_service.L7PolicyService(conf)
+        self.clean_wrapper_policy = True
```

Comment:
I see how clean_wrapper_policy is used at line 504, but how is this value set/unset?

Reply:
It's not.  I need to consider how this should be done.